<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CasparCG Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>6</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitterHorizontal">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QFrame" name="frameSplitterVertical">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>344</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>344</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="frameSplitterVerticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frameSplitterHorizontal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>288</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>288</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="frameSplitterHorizontalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetOnAirNow" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>360</width>
               <height>288</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>360</width>
               <height>288</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ClockWidget" name="widgetClock" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>288</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>288</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widgetOnAirNext" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>360</width>
               <height>288</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>360</width>
               <height>288</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetTimeline" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitterVertical">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>6</number>
       </property>
       <widget class="LibraryWidget" name="widgetLibrary" native="true">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="RundownWidget" name="widgetRundown" native="true">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="PreviewWidget" name="widgetPreview" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="InspectorWidget" name="widgetInspector" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>27</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClockWidget</class>
   <extends>QWidget</extends>
   <header>ClockWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>PreviewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InspectorWidget</class>
   <extends>QWidget</extends>
   <header>Inspector/InspectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibraryWidget</class>
   <extends>QWidget</extends>
   <header>Library/LibraryWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RundownWidget</class>
   <extends>QWidget</extends>
   <header>Rundown/RundownWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>addMedia()</slot>
  <slot>removeMedia()</slot>
  <slot>toggleFullscreen()</slot>
  <slot>toggleMaximize()</slot>
 </slots>
</ui>
